.github/workflows/_build.yml

```diff
@@ -208,7 +206,7 @@ jobs:
           retention-days: 5
 
       - name: Notify PR of screenshots failure
-        if: ${{ failure() }}
+        if: ${{ failure() && steps.screenshotTests.outputs.CHANGED_FILES != '' }}
```

Comment:
Ensure we only add the comment if screenshots actually changed as it's mighty confusing right now